### ISSUE TITLE
Android: Export: Godot Blue Background apear on exported apk.

### ISSUE THREAD

**Issue by llama-nl** [godotengine/godot#100513]
### Tested versions

Godot 4.3, 4.4+

### System information

Android 14 - Compatibility Renderer

### Issue description

In first splash screen (screen that apear before main splash screen) background color is blue. expecting it black or any way to customize the color. Sometimes it doesn't match with game theme 

Video:

https://github.com/user-attachments/assets/44f0d2bf-fc64-4d36-89b1-f1345c21df88

Screenshot:
![Image](https://github.com/user-attachments/assets/b54f1673-8911-4160-9f70-960ca1b16d5d)

### Steps to reproduce

Exported game to apk

### Minimal reproduction project (MRP)

Example App: https://play.google.com/store/apps/details?id=com.mw_games.play2race

**Comment by AThousandShips** [comment]
Please upload a project file, an exported project doesn't work as an MRP as it cannot be investigated or tested properly

**Comment by syntaxerror247** [comment]
The background for the Android native boot splash is the same as the Adaptive Icon background by default, which uses Godot's default icon background. To customize this, set the `Adaptive Icon Background` in the export settings to a PNG with a solid color of your choice.

**Comment by AThousandShips** [comment]
Marking this as something to look into documenting, thank you for clarifying!

**Comment by llama-nl** [comment]
> The background for the Android native boot splash is the same as the Adaptive Icon background, which is Godot's icon background by default. To customize this, set the `Adaptive Icon Background` in the export settings to a PNG with a solid color of your choice.

Yes, you are right. It fix the problem.  Few more things, when I give `launcher_icons/adaptive_foreground_432x432` it changes the application icon and ignore the `Main Icon`. I want to make my icon apear like godot apk.
example:
![Image](https://github.com/user-attachments/assets/1d7cc269-f9b2-4951-aaa9-ffa43fff98d0)
A solid color and the main icon top of the bg

**Comment by syntaxerror247** [comment]
> Few more things, when I give launcher_icons/adaptive_foreground_432x432 it changes the application icon and ignore the Main Icon

It is intentional. Fallback chain for Adaptive Icon Foreground `Provided foreground icon -> Provided main icon -> Project icon -> Default Godot foreground icon.`
See https://docs.godotengine.org/en/latest/tutorials/export/exporting_for_android.html#providing-launcher-icons

@llama-nl Keep this issue open, as we need to document that the Android native splash background and icon are the same as the Adaptive background and icon.
